### ISSUE TITLE
Support device pairing

### ISSUE THREAD

**Issue by janimo** [janimo/textsecure-qml#36]
Allow pairing with Signal desktop.

**Comment by StuntsPT** [comment]
I wanted to try [this](https://aur.archlinux.org/packages/signal/), but since I don't own and android device I was unable to scan the code that the app displays.
Is there any workaround for this? Thanks.
I can use "Tagger" to scan the QR code, but not sure what to do with the contents...
I suppose the app is missing the core functionality to work with it. Or is there some CLI way to work around the issue?

**Comment by janimo** [comment]
Right, we currently miss this functionality. Development has more or less stopped at the beginning of this year when it became apparent that there's no clear way to get push notifications working, without which the client is far from being really useful.

**Comment by StuntsPT** [comment]
So #12 is pretty much blocking every other issue...
I'll try to make some noise in the relevant upstream bugs, since there have been no news for a while now.

**Comment by janimo** [comment]
Thanks, but upstream have legitimate reasons for not working on Ubuntu support (no market for it, few resources to spare). Making noise is unlikely to help.

**Comment by StuntsPT** [comment]
Sorry, I wasn't clear:
I meant upstream Ubuntu-touch. The current lifecycle policies are **the** main community complaint regarding the platform. A plugin system for account-polld would greatly mitigate this issue as that also seems to be the main (but not the only) gripe with the policy.

**Comment by janimo** [comment]
Sure, progress in Ubuntu on this front would be great as probably other apps lack such functionality.
Not sure about your odds in speeding up the issue though, it has been a known issue for a long time AFAIK.
cheers
